socketmode/client.go

```diff
@@ -61,3 +61,8 @@ type Client struct {
 	debug bool
 	log   ilogger
 }
+
+// return a pointer to the slack API client
+func (c Client) GetApiClient() *slack.Client {
+	return &c.apiClient
```

Comment:
I did not know if it would have an unexpected effect on top of that `socketmode.apiClient` is not exported so it cannot be used outside of the package

Reply:
Sorry for the confusion. The following is my suggestion:

```go
type Client {
    ...
    apiClient *slack.Client
}

func (c Client) GetApiClient() *slack.Client {
  return c.apiClient
}
```